tools/grab_spotify_saved_tracks.py

```
@@ -0,0 +1,175 @@
+import logging
+from dotenv import load_dotenv
+from slugify import slugify
+import hashlib
+
+# Load environment variables
+load_dotenv()
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)